package.json

```diff
@@ -6,48 +6,24 @@
   "license": "agpl",
   "private": true,
   "scripts": {
-    "build": "NODE_ENV=production webpack --config webpack.prod.js",
-    "dev": "NODE_ENV=development webpack --config webpack.dev.js",
-    "watch": "NODE_ENV=development webpack --config webpack.dev.js --watch",
-    "deploy": "rm -rf js/* && npm install && npm run build && cd .. && tar -cvzf checksum.tar.gz -X checksum/.exclude checksum && cd checksum"
+    "build": "NODE_ENV=production webpack --config webpack.config.js",
+    "dev": "NODE_ENV=development webpack --config webpack.config.js",
+    "watch": "NODE_ENV=development webpack --config webpack.config.js --watch",
+    "deploy": "rm -rf js/* && npm install && npm run build && cd .. && tar -cvzf checksum.tar.gz -X checksum/.exclude checksum && cd checksum",
+    "lint": "eslint --ext .js,.vue --ignore-pattern tests src",
+    "lint:fix": "eslint --ext .js,.vue --ignore-pattern tests src --fix"
```

Comment:
We can also get rid of the linting config and dependencies. I tried running it and there are a lot of linter warnings and errors because it is not enforced on pull requests.

Let me know in case I should drop it.